src/Nix/Reduce.hs

```diff
@@ -113,7 +113,7 @@ staticImport pann path = do
 
 reduceExpr :: MonadIO m => Maybe FilePath -> NExprLoc -> m NExprLoc
 reduceExpr mpath expr
-    = (`evalStateT` M.empty)
+  = (`evalStateT` M.empty)
```

Comment:
This is unnecessary...

Reply:
Indeed, this was unintentional. I'll fix it tomorrow morning.

Reply:
done